### PR TITLE
Updating scale section

### PR DESCRIPTION
**Pull request by djodjo3**
Updating the Scale/Optimizations section with latest findings.

### CODE REVIEW

File: src/content/1.7/scale/optimizations.md [hunk 1]
```diff
@@ -32,10 +32,12 @@ There's no way to enumerate all the performance issues we've encountered, but le
 
 ### 1) PHP
 
+PHP tuning is very important for your application’s performance, either you’re running PrestaShop or any other PHP software.
```
Comment (eternoendless) [inline src/content/1.7/scale/optimizations.md:35]:
```suggestion
PHP tuning is very important for your application’s performance, whether you’re running PrestaShop or any other PHP software.
```

File: src/content/1.7/scale/optimizations.md [hunk 2]
```diff
@@ -97,12 +99,28 @@ Good news is, OPCache will not only store your PHP files in memory, but it will
...
+```
+
+If you can manage it, here are few options you could configure:
+
+```
+opcache.validate_timestamps=0
+opcache.revalidate_path=0
 ```
 
+Keep in mind if you deactivate `validate_timestamps` OPCache will never update your code unless you let him know explicitly (either through internal functions or by restarting the webserver).
```
Comment (eternoendless) [inline src/content/1.7/scale/optimizations.md:121]:
```suggestion
Keep in mind if you deactivate `validate_timestamps` OPCache will never update your code unless you let it know explicitly (either through internal functions or by restarting the web server).
```

File: src/content/1.7/scale/optimizations.md [hunk 3]
```diff
@@ -141,31 +159,59 @@ If you're using PHP-FPM, you should be able to use apache mpm_event. Using the f
 
 ### 5) MySQL/MariaDB Settings
 
-If you using MySQL < 8 or MariaDB, enable the query cache by putting this setting in the ```/etc/mysql/my.cnf``` file:
+Though we’re discussing it just now, MySQL tuning is just as important.
+
+Our intention here is to optimize the instance’s throughput by adding caches.
+
+As for PHP, it allows the service to work as much as possible in memory and avoid disk accesses, hence reducing latency.
```
Comment (eternoendless) [inline src/content/1.7/scale/optimizations.md:166]:
```suggestion
As for PHP, it allows the service to work as much as possible in memory and avoid disk access, hence reducing latency.
```

File: src/content/1.7/scale/optimizations.md [hunk 4]
```diff
@@ -141,31 +159,59 @@ If you're using PHP-FPM, you should be able to use apache mpm_event. Using the f
...
+
+#### Caching
+
+Those parameters allow better cache information for further reuse, first by enabling it, then by increasing its sizes.
```
Comment (eternoendless) [inline src/content/1.7/scale/optimizations.md:170]:
```suggestion
These parameters allow better cache information for further reuse, first by enabling it, then by increasing its size. 
```

File: src/content/1.7/scale/optimizations.md [hunk 5]
```diff
@@ -141,31 +159,59 @@ If you're using PHP-FPM, you should be able to use apache mpm_event. Using the f
...
+Our intention here is to optimize the instance’s throughput by adding caches.
+
+As for PHP, it allows the service to work as much as possible in memory and avoid disk accesses, hence reducing latency.
+
+#### Caching
+
+Those parameters allow better cache information for further reuse, first by enabling it, then by increasing its sizes. 
+Again, the idea is to keep the query results in memory rather than looking them up to the (higher latency) hard drive.
+
+As always, those values should be adapted to your own environment, you probably won’t need a ``host_cache_size`` of 1000.
```
Comment (eternoendless) [inline src/content/1.7/scale/optimizations.md:173]:
```suggestion
As always, these values should be adjusted to your own environment, you probably won’t need a ``host_cache_size`` of 1000. 
```

File: src/content/1.7/scale/optimizations.md [hunk 6]
```diff
@@ -141,31 +159,59 @@ If you're using PHP-FPM, you should be able to use apache mpm_event. Using the f
...
 ```
 
-Other important settings are:
+#### Buffering
+
+Buffering is almost another word for caching. 
+
+So we work here with the memory area that holds cached data for InnoDB tables, indexes, and other auxiliary buffers, etc..
+
+Again, those options should be adapted to your shop.
```
Comment (eternoendless) [inline src/content/1.7/scale/optimizations.md:190]:
```suggestion
Again, these options should be adapted to your shop. 
```

File: src/content/1.7/scale/optimizations.md [hunk 7]
```diff
@@ -141,31 +159,59 @@ If you're using PHP-FPM, you should be able to use apache mpm_event. Using the f
...
-Other important settings are:
+#### Buffering
+
+Buffering is almost another word for caching. 
+
+So we work here with the memory area that holds cached data for InnoDB tables, indexes, and other auxiliary buffers, etc..
+
+Again, those options should be adapted to your shop. 
+
+Setting up the ```innodb_buffer_pool_size``` to 1G maybe too much for your SQL instance, just make sure you have enough memory according to the value you configure.
```
Comment (eternoendless) [inline src/content/1.7/scale/optimizations.md:192]:
```suggestion
Setting up the `innodb_buffer_pool_size` to 1G may be too much for your SQL instance, just make sure you have enough memory according to the value you configure.
```

File: src/content/1.7/scale/optimizations.md [hunk 8]
```diff
@@ -141,31 +159,59 @@ If you're using PHP-FPM, you should be able to use apache mpm_event. Using the f
...
+
+The important thing is, if possible, to set ```innodb_buffer_pool_size``` to a greater value than your database size.
```
Comment (eternoendless) [inline src/content/1.7/scale/optimizations.md:194]:
```suggestion
The important thing is, if possible, to set `innodb_buffer_pool_size` to a greater value than your database size.
```

File: src/content/1.7/scale/optimizations.md [hunk 9]
```diff
@@ -141,31 +159,59 @@ If you're using PHP-FPM, you should be able to use apache mpm_event. Using the f
...
-tmp_table_size                        = 32M
-max_heap_table_size                   = 32M
-table_definition_cache                = 1000
-performance_schema                    = OFF
+read_buffer_size			= 2M 
+read_rnd_buffer_size			= 1M
+join_buffer_size			= 2M 
+sort_buffer_size 			= 2M
+innodb_buffer_pool_size 		= 1G
+
```
Comment (eternoendless) [inline src/content/1.7/scale/optimizations.md:203]:
```suggestion
read_buffer_size			= 2M 
read_rnd_buffer_size		= 1M
join_buffer_size			= 2M 
sort_buffer_size 			= 2M
innodb_buffer_pool_size 	= 1G

```

Comment (djodjo3) [reply src/content/1.7/scale/optimizations.md:203]: Strange, yours are not aligned anymore Oo
Am I missing anything ?

Comment (eternoendless) [reply src/content/1.7/scale/optimizations.md:203]: It looks well on my end 🤔 

Screenshot 2020-04-22 at 16 48 55

Comment (djodjo3) [reply src/content/1.7/scale/optimizations.md:203]: That's a funny one ^^

Capture d’écran 2020-04-23 à 00 51 17

**Comment by eternoendless** [comment]
Thank you @djodjo3 !

**Pull request merged**